.github/workflows/master-a380x.yml

```diff
@@ -0,0 +1,95 @@
+# Build script for the A380X on master branch
+# Build independent of the A32NX to avoid side effects if one or the other fails
+
+# TODO: Add a check if something changed outside of the fbw-a32nx folder (which isn't of interest for this build)
+#       Consider that master pre release should always have a A32NX and A380x though
+
+name: master
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    # Prevent running this on forks
+    if: github.repository_owner == 'flybywiresim'
+    runs-on: ubuntu-latest
+    env:
+      A32NX_PRODUCTION_BUILD: 1
+      A32NX_INSTRUMENTS_BUILD_WORKERS: 2
```

Comment:
Do we need to use the environment variables with the A32NX prefix here or may we either introduce A380X specific environment variables or change these ones to a more general name?

Reply:
I renamed `A32NX_PRODUCTION_BUILD` to `FBW_PRODUCTION_BUILD`. Removed `A32NX_INSTRUMENTS_BUILD_WORKERS` as it's no longer used.